packages/build-essentials/src/webpack.config.js

```diff
@@ -33,8 +33,8 @@ const webpackConfig = {
                 loader: 'json-loader'
             },
             {
-                test: /\.(woff|woff2|eot|svg)$/,
-                loader: 'url-loader?limit=100000'
+                test: /\.(woff|woff2)$/,
```

Comment:
Because you are still using it. The loader "url?limit=100000" is a shortcut for "url-loader?limit=100000". Webpack automatically appends "-loader" to it when the first module lookup failed. Maybe it's still working because another package has the module "url-loader" as a dependency.
Anyway I would recommend to not use the short syntax because it can lead to strange issues if there is another module called "url". Therefore this logic has been removed in Webpack 2 by default. (see https://github.com/webpack/webpack/issues/2986)